### DISCUSSION TITLE
Plotting of reconstructed pole figure - Unit of z-axis?

### DISCUSSION THREAD

**Discussion by Cumarin8314** [mtex-toolbox/mtex#1838] [Ask Anything]
Hello!

I have measured in-plane pole figures of thin films with varying degrees of texture, which I want to plot nicely. Since my understanding of crystallography is not that deep, I hope that you could resolve some of my open questions, which can not be fully resolved by reading the documentation. At the end, I have pasted matlab code.

1) I have plotted the pole figures reconstructed from the ODF. For the pole figure, the intensity ranges from 0 to 3.5, which the exact range of the ODF itself. Does that mean the intensity of the reconstructed pole figures represents multiples of random distribution as well and thus can be used to compare varying degrees of textures or is it dimensionless?

2) I have measured the pole figure are the position of the (005) peak of a crystal (Ge2Sb2Te5) in the space group P-3m1 (No 164). Is the way I have specified the symmetry correct? 

Many thanks in advance!

```
%% Specify Crystal and Specimen Symmetries

% crystal symmetry
CS = crystalSymmetry('P-3m1');

% specimen symmetry
SS = specimenSymmetry('1');

% plotting convention
setMTEXpref('xAxisDirection','north');
setMTEXpref('zAxisDirection','outOfPlane');

%% Specify File Names

% path to files
pname = 'Test';

% which files to be imported
fname = [pname '\GST225.txt'];

%% Specify Miller Indice

h = { ...
  Miller(0,0,5,CS),...
  };

%% Specifiy Structural Coefficients for Superposed Pole Figures

c = {1};

%% Import the Data

% create a Pole Figure variable containing the data
pf = PoleFigure.load(fname,h,CS,SS,'superposition',c,'interface','rigaku');

odf = calcODF(pf,'silent')

plotPDF(odf,pf.h,'antipodal')
%plotDiff(pf,odf)
caxis([0 3.8])
mtexColorMap viridis
mtexColorbar
```

**Comment by kilir** [comment]
Hi,

wrt 1) An odf from one pole figure is ambitious, but as long as you only look at the (001) that really doesn't matter. Units are multiples of uniform. You can compare those, as long as all paramters for contructing the density function are kept identical. Other measures such as the norm()/sqrt(4*pi) of the smooth pole figure 
or better the volume percentage within a given cone might often be more robust.

wrt 2) since you are only interested in the 001 pole figure, your symmetry should be fine. If at some point you measure and consider other crystal directions, you have to consider the appropriate lattice parameters as well as convention.

Cheers,
Rüdiger
